readme.md

```diff
@@ -467,6 +467,7 @@
 
 ## Work
 
+- [Cisco Spark](https://github.com/CiscoDevNet/awesome-ciscospark) - Calls, Video & Messaging for Teams collaboration.
```

Comment:
The repo name would be better as `awesome-cisco-spark` to make it clear it's two words.

Reply:
hi @sindresorhus, 

thanks for the review

good points: fixed
- license, 
- Node.js mentions
- awesome badge

for the repo name, I get your point.
yet, the 'awesome-ciscospark' spelling makes more sense for this community, as all github & web resources for "Cisco Spark" are spelled 'ciscospark':
- www.ciscospark.com
- developer.ciscospark.com
- https://github.com/ciscospark/react-ciscospark